holmes/core/toolset_manager.py

```
@@ -0,0 +1,394 @@
+    toolsets: Optional[dict[str, dict[str, Any]]]
+    custom_toolsets: Optional[List[FilePath]]
+    custom_toolsets_from_cli: Optional[List[FilePath]]
+
+    toolset_status_location: FilePath
+
+    def __init__(
+        self,
+        toolsets: Optional[dict[str, dict[str, Any]]] = None,
+        custom_toolsets: Optional[List[FilePath]] = None,
+        custom_toolsets_from_cli: Optional[List[FilePath]] = None,
+        toolset_status_location: FilePath = FilePath(DEFAULT_TOOLSET_STATUS_LOCATION),
```

Ruff (0.11.9)

Do not perform function call `FilePath` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)